tvp_go19_db_test.go

```diff
@@ -678,7 +678,7 @@ func TestTVPSchema(t *testing.T) {
 	}
 
 	tvpType := TVP{
-		TypeName: "exempleTVP",
+		TypeName: "TestTVPSchema.exempleTVP",
```

Comment:
Why do you change type name here?

Reply:
The tvp implementation changed from having 3 fields (TVPTypeName, TVPSchema, and TVPValue) to having only 2 fields (TypeName and Value) (see PR #453 ). The `TypeName` should include the schema which the tvp belongs to if applicable. When this test was updated, the `TVPSchema` field was removed, but the `TypeName` was not updated.